dotNETStandard/HealthVaultProviderManagementPortal/HealthVaultProviderManagementPortal/Controllers/GoalsController.cs

```diff
@@ -30,26 +30,32 @@ public class GoalsController : Controller
         /// </summary>
         public async Task<ActionResult> Index(Guid personId, Guid recordId)
         {
-            var response = await ExecuteMicrosoftHealthVaultRestApiAsync(api => api.GetGoalsAsync(), personId, recordId);
+            var response = await ExecuteMicrosoftHealthVaultRestApiAsync(api => api.Goals.GetActiveAsync(), personId, recordId);
             return View(response);
         }
 
         /// <summary>
         /// Get a goal for the logged in user
         /// </summary>
         [HttpGet]
-        public async Task<ActionResult> Goal(Guid id, Guid personId, Guid recordId)
+        public async Task<ActionResult> Goal(Guid? id, Guid personId, Guid recordId)
```

Comment:
Specifying the corresponding attribute should resolve the ambiguity, ie: `[HttpGet]`, `[HttpPost]`, etc.